lib/config/providers/chat_provider.dart

```diff
@@ -348,6 +352,40 @@ class ChatNotifier extends Notifier<ChatState> {
     return state.getUnreadCountForGroup(groupId);
   }
 
+  Future<void> refreshUnreadCount(String groupId) async {
+    try {
+      final activePubkey = ref.read(activePubkeyProvider) ?? '';
+      if (activePubkey.isEmpty) {
+        return;
+      }
+
+      final lastRead = await LastReadService.getLastRead(
+        groupId: groupId,
+        activePubkey: activePubkey,
+      );
+
+      final messages = state.groupMessages[groupId] ?? [];
+      final baseline = lastRead ?? DateTime.fromMillisecondsSinceEpoch(0);
```

Comment:
[sug] Maybe this could be the default value for getLastRead method in `LastReadService`?

Reply:
ok!